site/docs/v1/tech/apis/system.adoc

```diff
@@ -283,6 +283,11 @@ include::docs/src/json/logger/response.json[]
 
 The Status API is used to retrieve the current status and metrics for FusionAuth. This is useful for health checks and monitoring.
 
+[NOTE]
+====
+FusionAuth also supports a system status check using Prometheus.  See Prometheus documentation here.
```

Comment:
"See Prometheus documentation here" needs a link. 

I've also been hiding links to unreleased documentation by using the `//` string which hides the entire line in asciidoctor. So you could change this to:

```
FusionAuth also supports a system status check using Prometheus.  
// TBD add link when promethus docs are live. See Prometheus documentation here.
```

and then we can grep through the docs for the string `TBD`.